template/razor.go

```diff
@@ -186,18 +187,21 @@ func (t *Template) ensureInit() {
 }
 
 func printDebugInfo(r replacement, content string) {
-	if r.name == "" {
+	if r.name == "" || !InternalLog.IsLevelEnabled(logrus.DebugLevel) {
```

Comment:
This is always going to be true. Logrus is always set to `TRACE`. You would need to implement a function in multilogger

Reply:
Done